examples/basics/semantic_segmentation/main.py

```diff
@@ -33,12 +33,10 @@
 
 # activate depth rendering
 bproc.renderer.enable_depth_output(activate_antialiasing=False)
+bproc.renderer.enable_semantic_segmentation(map_by=["class", "instance", "name"])
```

Comment:
I would call it `enable_segmentation_output()`. [Semantic segmentation](https://en.wikipedia.org/wiki/Image_segmentation#Groups_of_image_segmentation) relates to class-specific segmentation without distinguishing instances. Also to align with the other options.

Reply:
Good point. I will change it.